vm-setup/roles/v1aX_integration_test/vars/main.yml

```diff
@@ -65,6 +66,16 @@ IMAGE_NAME_CENTOS: "{{ lookup('env', 'IMAGE_NAME') | default('CENTOS_8.2_NODE_IM
 RAW_IMAGE_NAME_CENTOS: "{{ lookup('env', 'IMAGE_RAW_NAME') | default('CENTOS_8.2_NODE_IMAGE_K8S_v1.18.8-raw.img', true)}}"
 IMAGE_LOCATION_CENTOS: 'https://artifactory.nordix.org/artifactory/airship/images/k8s_v1.18.8/'
 
+# Environment variables to install Ironic with TLS and Basic Auth
+GOPATH: "{{ lookup('env', 'GOPATH')}}"
+IRONIC_TLS_SETUP: "{{ lookup('env', 'IRONIC_TLS_SETUP') | default('true', true)}}"
+IRONIC_BASIC_AUTH: "{{ lookup('env', 'IRONIC_BASIC_AUTH') | default('true', true)}}"
+DEPLOY_BMO: "{{ lookup('env', 'DEPLOY_BMO') | default('false', true)}}"
+DEPLOY_IRONIC: "{{ lookup('env', 'DEPLOY_IRONIC') | default('true', true)}}"
```

Comment:
Those two should not be set as variables, they have to be hardcoded values depending on what you do

Reply:
Addressed in #516